xlive/Blam/Engine/interface/user_interface.h

```diff
@@ -1,300 +1,10 @@
 #pragma once
+#include "user_interface_shared_globals.h"
+#include "saved_games/game_variant.h"
+#include "saved_games/saved_game_files.h"
```

Comment:
remove saved game includes and put `struct` infront of the parameter and return types in the prototypes below